CONTRIBUTING.md

```diff
@@ -32,15 +34,58 @@ Basically we have four big directories with different purposes:
 | **`develop/`** | This is where we _explore_ data, feel free to create your own notebook for your exploration. | `[ISO 8601 date]-[author-initials]-[2-4 word description].ipynb` (e.g. `2016-05-13-ec-air-tickets.ipynb`) |
 |**`report/`** | This is where we write up the findings and results, here is where we put together different data, analysis and strategies to make a point, feel free to jump in. | Meaninful title for the report (e.g. `Transport-allowances.ipybn` |
 | **`src/`** | This is where our auxiliary scripts lies, code to scrap data, to convert stuff etc. | Small caps, no special character, `-` instead of spaces. |
-| **`data/`** | This is not suppose to be commit, but it is where saved databases will be stored locally (scripts from `src/` should be able to get this data for you); a copy of this data will be available elsewhere (_just in case_…). | Small caps, no special character, `-` instead of spaces. |
+| **`data/`** | This is not suppose to be committed, but it is where saved databases will be stored locally (scripts from `src/` should be able to get this data for you); a copy of this data will be available elsewhere (_just in case_). | Small caps, no special character, `-` instead of spaces. |
+
+### Source files (`src/`)
+
+Here we explain what each script from `src/` does for you:
+
+##### One script to rule them all
+
+1. `src/fetch_datasets.py` dowloads all the available datasets to `data/` is `.xz` compressed CSV format with headers translated to English.
+
+
+##### Quota for Exercising Parliamentary Activity (CEAP)
+
+1. `src/fetch_datasets.py --from-source` dowloads all CEAP datasets to `data/` from the oficial source (in XML format in Portuguese) .
+1. `src/fetch_datasets.py` dowloads the CEAP datasets into `data/`; it can download them from the oficial source (in XML format in Portuguese) or from our backup server (`.xz` compressed CSV format, with headers translated to English).
+1. `src/xml2csv.py` converts the original XML datasets to `.xz` compressed CSV format.
+1. `src/translate_datasets.py` translates the datasets file names and the labels of the variables within these files.
+1. `translation_table.py` creates a `data/YYYY-MM-DD-ceap_datasets.md` file with details of the meaning and of the translation of each variable from the _Quota for Exercising Parliamentary Activity_ datasets.
```

Comment:
From this line you refer `src` scripts as if were in the root path. Can you correct before merging?